server/routes/addPrisoner/prisonerDetailsController.ts

```diff
@@ -87,35 +87,37 @@ export default class PrisonerDetailsController {
       body('firstName', 'Enter a first name').trim().isLength({ min: 1, max: 250 }),
       body('lastName', 'Enter a last name').trim().isLength({ min: 1, max: 250 }),
 
-      body(['day', 'month', 'year']).trim(),
-      // 'prisonerDob' not an actual form field but used to store combined result and any validation errors
-      body('prisonerDob')
-        .customSanitizer((_value, { req }) => {
-          const day = Number.parseInt(req.body?.day, 10).toString().padStart(2, '0')
-          const month = Number.parseInt(req.body?.month, 10).toString().padStart(2, '0')
-          const year = Number.parseInt(req.body?.year, 10).toString()
-          return `${year}-${month}-${day}`
-        })
-        .custom((date: string) => {
-          if (date === 'NaN-NaN-NaN') {
-            throw new Error('Enter a date of birth')
-          }
-
-          if (date.includes('NaN')) {
-            throw new Error('Enter a date of birth and include a day, month and year')
-          }
-
-          const parsedDate = parseISO(date)
-          if (!isValid(parsedDate)) {
-            throw new Error('Date of birth must be a real date')
-          }
-
-          if (isAfter(parsedDate, new Date())) {
-            throw new Error('Date of birth must be in the past')
-          }
-
-          return true
-        }),
+      // Prisoner date of birth
+      body(['prisonerDob-day', 'prisonerDob-month', 'prisonerDob-year']).trim(),
+      // 'prisonerDob' not an actual form field but used to store combined result
+      body('prisonerDob').customSanitizer((_value, { req }) => {
+        const day = Number.parseInt(req.body?.['prisonerDob-day'], 10).toString().padStart(2, '0')
+        const month = Number.parseInt(req.body?.['prisonerDob-month'], 10).toString().padStart(2, '0')
+        const year = Number.parseInt(req.body?.['prisonerDob-year'], 10).toString()
+        return `${year}-${month}-${day}`
+      }),
+      // set any validation errors against the 'day' field so ErrorSummary links to this
+      body('prisonerDob-day').custom((_value, { req }) => {
```

Comment:
Key change here: return the combined `YYY-MM-DD` field as `prisonerDob` but accumulate any errors against just the 'day' input so this is what the error summary links to.